tapir/coop/views/draftuser.py

```diff
@@ -103,6 +103,17 @@ class DraftUserDetailView(
     permission_required = PERMISSION_COOP_MANAGE
     model = DraftUser
 
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        draft_user: DraftUser = self.object
+        context_data["similar_members"] = ShareOwner.objects.filter(
+            Q(last_name=draft_user.last_name)
```

Comment:
How common is the last name between the users? I guess it could be a false positive quite often, but maybe still not too often.

Reply:
Yes we'll probably get false positives regularly, but then the Modal shows more details about both members, so it should be clear for the user if it's a false positive or not : https://github.com/SuperCoopBerlin/tapir/blob/159ba34945b98a475b7bdde46735928d983ccfa3/tapir/coop/templates/coop/draftuser_detail.html#L69-L92